app/views/categories/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= render "shared/nav_bar"%>
+<%= render "shared/messages"%>
+<div class="ms-4 mt-2 w-50">
+<h2>Add a category</h2>
+<%= form_with model: @category, url: categories_path do |f|%>
+<div class="mt-3">
+  <%= f.label :name, class: 'form-label' %>
+  <%= f.text_field :name, class: 'form-control' %>
+  </div>
+  <div class="mt-3" >
+  <%= f.label :icon_url, class: 'form-label' %>
+  <%= f.text_field :icon_url, class: 'form-control' %>
+  </div>
+
+  <%= f.submit "Save", class: 'mt-2' %>
+<%end%>
+<% if flash[:error] %>
+    <div><%= flash[:error] %></div>
+<% end %>
+</div>
```

Comment:
- [ ] The forms should mostly be the same as we have in the design. Kindly maintain the colors and the position of the submit button (top-right).

- [ ] Kindly ensure you also maintain the styling of the input forms. As you can see they stretch from end to end without border-radius.

  Your Implementation | Behance Design
  :---:|:---:
  ![djww](https://user-images.githubusercontent.com/36542986/220813982-86ed7f10-b27f-4463-bae8-3ced6e3d5027.jpg) | ![siow](https://user-images.githubusercontent.com/36542986/220813986-f66ad349-ab0a-4924-aceb-c4e463a45080.jpg)


![image](https://user-images.githubusercontent.com/36542986/220813615-c7203d10-9023-4c19-b812-cb9c33c4b17c.png)